### PR TITLE
Remove per-page UI and use 50 in all paginated lists 50 in admin

### PR DESCRIPTION
**Pull request by h-lame**
For: https://www.pivotaltracker.com/story/show/97713802

### CODE REVIEW

File: app/controllers/admin/petitions_controller.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def show
   end
 
   def threshold
-    @petitions = Petition.threshold.order(:signature_count).paginate(:page => params[:page], :per_page => params[:per_page] || 20)
+    @petitions = Petition.threshold.order(:signature_count).paginate(:page => params[:page], :per_page => 50)
```
Comment (otlaitil) [inline app/controllers/admin/petitions_controller.rb]: Would we gain anything by always using our custom search? It includes the pagination and the implementation would be uniform (`@petitions` would always be an instance of `Browseable:Search` everywhere - now it's mixed with `ActiveRecord:Relation`).

**Comment by otlaitil** [comment]
LGTM :+1:

File: app/controllers/admin/petitions_controller.rb (hunk 1 above)
Comment (davidbasalla) [inline app/controllers/admin/petitions_controller.rb]: What Otto said, although this method will disappear once my branch https://github.com/alphagov/e-petitions/pull/212 gets merged in (removing threshold and todolist).

**Comment by mybits** [comment]
LGTM :+1:

**Comment by davidbasalla** [comment]
looks good

**Pull request merged**